bpf/lib/srv6.h

```diff
@@ -143,7 +143,7 @@ srv6_encapsulation(struct __ctx_buff *ctx, int growth, __u16 new_payload_len,
 
 	/* Add room between Ethernet and network headers. */
 	if (ctx_adjust_hroom(ctx, growth, BPF_ADJ_ROOM_MAC,
-			     ctx_adjust_hroom_flags()))
+			     BPF_F_ADJ_ROOM_ENCAP_L3_IPV6))
```

Comment:
Which kernel version does this impose.

Reply:
It was from 5.2 https://github.com/torvalds/linux/commit/868d523535c2d00b696753ece606e641a816e91e.